TestCoin.ts

```diff
@@ -0,0 +1,24 @@
+import { Coin } from '../types';
+import { randomBytes } from 'crypto';
+
+export class TestCoin {
+  /**
+   * Creates a single test coin with given parameters
+   */
+  static create(params: Partial<Coin> = {}): Coin {
+    return {
+      id: params.id || `0x${randomBytes(32).toString('hex')}`,
+      owner: params.owner || `0x${randomBytes(32).toString('hex')}`,
```

Comment:
```suggestion
      id: params.id || getRandomB256(),
      owner: params.owner || getRandomB256(),
```

Favour helper functions